collect_app/src/main/res/layout/instance_map_layout.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <TextView
+        android:id="@+id/form_title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="start"
+        android:padding="4dp"
+        android:background="#BF000000"
+        android:textColor="#FFFFFF" />
+
+    <TextView
+        android:id="@+id/geometry_status"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:gravity="start"
+        android:padding="4dp"
+        android:background="#BF000000"
+        android:textColor="#FFFFFF" />
+
+    <FrameLayout
+        android:id="@+id/map_container"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@id/form_title"
+        android:layout_above="@id/geometry_status" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/new_instance"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="5dp"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentEnd="true"
+        android:layout_above="@id/geometry_status"
+        app:srcCompat="@drawable/ic_note_add"
+        android:padding="15dp" />
+
+    <ScrollView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_below="@+id/form_title"
+        android:layout_above="@+id/new_instance">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <androidx.appcompat.widget.AppCompatImageButton
```

Comment:
These should really be [Floating Action Buttons](https://material.io/components/buttons-floating-action-button/#anatomy). The Android implementation is documented [here](https://material.io/develop/android/components/floating-action-button/). Using those should give you the correct colors I believe (`colorOnPrimary` and `colorPrimary`).

Reply:
Agreed. @zestyping and I talked about this and thought that we'd initially match the other map views and then rethink their look and feel together.

Reply:
Yeah that does seem right.